Source/Modules/ModuleInput.cpp

```diff
@@ -17,6 +17,11 @@
 	#include "optick.h"
 #endif // DEBUG
 
+namespace
+{
+	const float COOLDOWN_SECONDS = 0.25f;
```

Comment:
```suggestion
const float COOLDOWN_SECONDS = 0.25f;
```
`Ctrl+K`, `Ctrl+D` should automatically fix this.